packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/di/app_dependencies.dart

```diff
@@ -142,11 +149,28 @@ class AppDependencies {
           create: (context) => AuthInterceptor(
             context.read(),
             context.read(),
-            context.read(),
+            context.read,
           ),
         ),
         Provider<AnalyticsInterceptor>(
           create: (context) => AnalyticsInterceptor(context.read()),
         ),
+        Provider<LogInterceptor>(
+          create: (context) => LogInterceptor(
+            request: false,
+            requestHeader: false,
+            requestBody: false,
+            responseHeader: false,
+            responseBody: false,
+            error: true,
+            logPrint: (object) {
```

Comment:
For better readability can we extract this anonymous function as a private method ?